TreePiece.cpp

```diff
@@ -1483,9 +1484,8 @@ void TreePiece::kick(int iKickRung, double dDelta[MAXRUNG+1],
       if(p->rung >= iKickRung) {
 	  if(bNeedVPred && TYPETest(p, TYPE_GAS)) {
 	      if(bClosing) { // update predicted quantities to end of step
-		  p->vPred() = p->velocity
-		      + dDelta[p->rung]*p->treeAcceleration;
-		  glassDamping(p->vPred(), dDelta[p->rung], dGlassDamper);
+		      p->vPred() = p->velocity + dDelta[p->rung]*p->treeAcceleration;
```

Comment:
Get the white spaces to match for these two lines, so that this doesn't look like a change.